src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -192,6 +192,27 @@ def test_download_linux_web_log(self, resource_group):
         self.assertTrue(os.path.isdir(os.path.join(log_dir, 'LogFiles', 'kudu', 'trace')))
 
 
+class WebappZipDeployScenarioTest(ResourceGroupVCRTestBase):
+    def __init__(self, test_method):
```

Comment:
can you use the new style by inheriting from ScenarioTest?
`AppServicePlanScenarioTest` right below this is a good example

Reply:
Reference: https://github.com/Azure/azure-cli/blob/dev/doc/scenario_base_tests.md

Reply:
the reason i used the older test style is because the new scenario tests cannot be viewed on VS 2017 test explorer, which makes debugging a problem.  i can add a comment here to update this to the new style once this issue is fixed.

Reply:
No. We will not approve new tests in the old form. As a matter of fact, we're moving all tests to new test base in near future. You can run test case through command line. You can also execute script directly by adding the following snippet to the end of the test file.,
```
if __name__ == '__main__':
    import unittest
    unittest.main()
```
There isn't a timeline for VS python extension to fix their issue. Dev tooling shouldn't impact project itself.